test/attachto.js

```diff
@@ -33,9 +34,9 @@ describe('attachTo', function() {
         attachTo(elm, h('div#attached', 'New text')),
       ]),
     ]);
-    patch(vnode0, vnode1);
+    elm = patch(vnode0, vnode1).elm;
```

Comment:
You create another node whenever "sel" is different than the previous vnode one. Since the starting node is just "div" when you start from a dom node, you have a root node re-creation in most tests due to a class or a different tagName in the new vnode.

Before my work, the root node was never re-created, so we missed the ability to change either its tagName, class or id, when specified in "sel". That's fixed in my implementation but I needed to fix some tests too to make it work.

Hope I'm clear.